.github/workflows/pull_request.yml

```diff
@@ -128,8 +128,23 @@ jobs:
       - name: Pytest
         env:
           NEBULA_TEST_LOGS_DIR: ${{ github.workspace }}/build
-        run: ./ci/test.sh test
+        run: ./ci/test.sh test --rm_dir=false
         timeout-minutes: 25
+      - name: Graph Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-graphd.ERROR && exit 1
+      - name: Meta Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-metad.ERROR && exit 1
+      - name: Storage Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-storaged.ERROR && exit 1
```

Comment:
we don't need these steps when having following upload-artifact action.

Reply:
It's not easy to download the logs and view it, so I print to screen for easy to check.

Reply:
Dumping lots of logs on the screen doesn't make it easier to read and let our ci run more time.

Reply:
Make sense. Fixed.